install/auto_installer.sh

```diff
@@ -0,0 +1,150 @@
+echo "Auto Installer works on any Ubuntu 16.04 server setup."
+echo "======================================================"
+read -n 1 -s -r -p "Press any key to start the installation."
+echo ""
+
+# Updating & Upgrading
+
+sudo apt-get update && sudo apt-get -y upgrade
+
+echo "======================================================"
+echo "Select **LAMP server** with space and then follow prompts."
+echo "======================================================"
+read -n 1 -s -r -p "Press any key to start"
+echo ""
+
+sudo apt-get -y install tasksel && sudo tasksel
+
+# Installing nano instead of vim
+
+sudo apt-get -y install nano
+
+# Installing required PHP extensions
+
+sudo apt-get install php-curl php-pear php-mbstring
+
+# Making sure curl is installed
+
+sudo apt-get -y install curl
+
+# Install Composer
+
+curl -sS https://getcomposer.org/installer | php
+sudo mv composer.phar /usr/local/bin/composer
+
+# Make /var/www/ writable.
+
+sudo chown -R $(whoami):$(whoami) /var/www/
+cd /var/www/
+
+# Install git and clone the repo.
+
+sudo apt-get install -y git
+git clone https://github.com/Nakiami/mellivora.git
+
+# Fetch required dependencies using Composer
+
+cd /var/www/mellivora/
+composer install
+
+# Copy and edit configuration file.
+
+cp /var/www/mellivora/include/config/config.inc.php.example /var/www/mellivora/include/config/config.inc.php
+cp /var/www/mellivora/include/config/db.inc.php.example /var/www/mellivora/include/config/db.inc.php
+
+echo "======================================================"
+read -n 1 -p "Would you like to edit mellivora config.inc.php file? (y/n) " ans;
+echo ""
+case $ans in
+    y|Y)
+        sudo nano /var/www/mellivora/include/config/config.inc.php;;
+    n|N)
+	echo ""
+	echo ""
+	echo "Ok, if you wish to modify it later the location is:"
+	echo "/var/www/mellivora/include/config/config.inc.php"
+	read -n 1 -s -r -p "Press any key to continue"
+        ;;
+    *)
+        exit;;
+esac
+
+# Make the writable directory writable.
+
+sudo chown -R www-data:www-data /var/www/mellivora/writable/
+
+# Copy and edit the Apache2 config file.
+
+sudo cp /var/www/mellivora/install/mellivora.apache.conf /etc/apache2/sites-available/mellivora.conf
+
+read -n 1 -p "Would you like to edit Apache2 config file? (y/n) " ans;
+
+case $ans in
+    y|Y)
+        sudo nano /etc/apache2/sites-available/mellivora.conf;;
+    n|N)
+	echo ""
+	echo ""
+	echo "Ok, if you wish to modify it later the location is:"
+	echo "/etc/apache2/sites-available/mellivora.conf"
+	read -n 1 -s -r -p "Press any key to continue"
+        ;;
+    *)
+        exit;;
+esac
+
+
+# Disable the default Apache site. Activate Mellivora. Restart Apache.
+
+sudo a2dissite 000-default
+sudo a2ensite mellivora
+sudo service apache2 restart
+
+
+# Create the Mellivora database and import the provided structure.
+
+echo "CREATE DATABASE mellivora CHARACTER SET utf8 COLLATE utf8_general_ci;" | mysql -u root -p
```

Comment:
These three MySQL commands will just send up a prompt asking for a password with no explanation of what password it's looking for, like
```
Enter password:
Enter password:
Enter password:
```
Would be nice if there was some echo before this explaining what was happening.

Reply:
That is the root MySQL password right?

Reply:
It is, yeah